### PR TITLE
nfd-master: implement --prune flag

### PR DESCRIPTION
**Pull request by marquiz**
A new sub-command like flag for cleaning up a cluster. When `--prune` is
specified nfd-master removes all NFD related labels, annotations and
extended resources from all nodes of the cluster and exits.

Also, add new `--kubeconfig` flag. Useful with `--prune` and for development purposes.

### CODE REVIEW

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

File: cmd/nfd-master/main.go [hunk 1]
```diff
@@ -63,15 +63,20 @@ func argsParse(argv []string) (master.Args, error) {
...
   %s -h | --help
   %s --version
 
   Options:
   -h --help                       Show this screen.
   --version                       Output version and exit.
+  --prune                         Prune all NFD related attributes from all nodes
+                                  of the cluster and exit.
+  --kubeconfig=<path>             Kubeconfig to use [Default: ]
+                                  of the cluster and exit.
```
[the remark this answers is not in the document]
Comment (marquiz) [reply cmd/nfd-master/main.go:79]: Thanks @Ethyling for the review! Now fixed

**Comment by marquiz** [comment]
Rebased and addressed the comment from @Ethyling

**Pull request merged**